# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpcom/base/MemoryTelemetry.cpp
# Commit: 60cccadbeaf0
# Full Hash: 60cccadbeaf01eee779400d8ae6f0c6e5f80da1c
# Author: Gabriele Svelto <gsvelto@mozilla.com>
# Date: 2019-11-01 16:13:11
# Regressor Bug: 1586236
# File Overlap Count: 1
# Description:
#   Bug 1586236 - Use memory resource notifications to detect low memory scenarios on Windows; r=dmajor
#   
#   This patch uses the low memory resource notification facility to detect
#   scenarios where physical memory is running low without polling. This is a
#   significant change compared to the previous behavior which measured both
# ==============================================================================

diff -r 23c113d65b48 -r 60cccadbeaf0 xpcom/base/MemoryTelemetry.cpp
--- a/xpcom/base/MemoryTelemetry.cpp	Fri Nov 01 06:03:35 2019 +0000
+++ b/xpcom/base/MemoryTelemetry.cpp	Thu Oct 31 13:55:42 2019 +0000
@@ -286,10 +286,6 @@
          UNITS_BYTES);
   RECORD(MEMORY_STORAGE_SQLITE, StorageSQLite, UNITS_BYTES);
 #ifdef XP_WIN
-  RECORD(LOW_MEMORY_EVENTS_VIRTUAL, LowMemoryEventsVirtual,
-         UNITS_COUNT_CUMULATIVE);
-  RECORD(LOW_MEMORY_EVENTS_COMMIT_SPACE, LowMemoryEventsCommitSpace,
-         UNITS_COUNT_CUMULATIVE);
   RECORD(LOW_MEMORY_EVENTS_PHYSICAL, LowMemoryEventsPhysical,
          UNITS_COUNT_CUMULATIVE);
 #endif